extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -127,20 +131,44 @@ public function getTopAdminsData() {
 		// Add details to top admins
 		$topAdminsTemplateData[ CommunityPageSpecialTopAdminsFormatter::TOP_ADMINS_LIST ] =
 			$this->getContributorsDetails(
-				$topAdminsTemplateData[ CommunityPageSpecialTopAdminsFormatter::TOP_ADMINS_LIST ]
+				$topAdminsTemplateData[ CommunityPageSpecialTopAdminsFormatter::TOP_ADMINS_LIST ],
+				AvatarService::AVATAR_SIZE_MEDIUM
 			);
 
 		$templateMessages = [
 			'topAdminsHeaderText' => $this->msg( 'communitypage-admins' )->text(),
 			'otherAdmins' => $this->msg( 'communitypage-other-admins' )->text(),
 			'noAdminText' => $this->msg( 'communitypage-no-admins' )->text(),
 			'noAdminContactText' => $this->msg( 'communitypage-no-admins-contact' )->text(),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "noAdminContactText" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "communitypage-no-admins-contact" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)